MediaBrowser.Model/Dto/BaseItemDto.cs

```diff
@@ -85,11 +85,6 @@ public class BaseItemDto : IHasProviderIds, IItemDto, IHasServerId
 
         public string PreferredMetadataCountryCode { get; set; }
 
-        /// <summary>
-        /// Gets or sets a value indicating whether [supports synchronize].
-        /// </summary>
-        public bool? SupportsSync { get; set; }
```

Comment:
@thornbill the web client is the only repository that actually uses this property but I believe the code using it is legacy:

https://github.com/jellyfin/jellyfin-web/blob/e7c149dd93f6195f83fc7c596f98672bfbfda6d3/src/components/itemHelper.js#L157C17-L167

https://github.com/jellyfin/jellyfin-web/blob/master/src/components/itemContextMenu.js#L293-L299

Reply:
Yeah looks like it 👍

Reply:
https://github.com/jellyfin/jellyfin-web/pull/5082